x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_indices/search_indices.tsx

```diff
@@ -114,22 +148,72 @@ export const SearchIndices: React.FC = () => {
     </EuiLinkTo>
   );
 
-  return (
-    <EnterpriseSearchContentPageTemplate
-      pageChrome={baseBreadcrumbs}
-      pageViewTelemetry="Search indices"
-      isLoading={false}
-      pageHeader={{
-        pageTitle: i18n.translate(
-          'xpack.enterpriseSearch.content.searchIndices.searchIndices.pageTitle',
+  const engineSteps = (
+    <>
+      <EuiTitle>
+        <h2>
+          {i18n.translate('xpack.enterpriseSearch.content.searchIndices.searchIndices.stepsTitle', {
+            defaultMessage: 'Build beautiful search experiences with Enterprise Search',
+          })}
+        </h2>
+      </EuiTitle>
+      <EuiSpacer size="l" />
+      <EuiFlexGroup>
+        <EuiFlexItem>
+          <GettingStartedSteps step={searchIndices.length === 0 ? 'first' : 'second'} />
+        </EuiFlexItem>
+        <EuiFlexItem grow={false}>
+          <ElasticsearchResources />
+        </EuiFlexItem>
+      </EuiFlexGroup>
+    </>
+  );
+
+  const pageTitle =
+    searchIndices.length !== 0
+      ? i18n.translate('xpack.enterpriseSearch.content.searchIndices.searchIndices.pageTitle', {
+          defaultMessage: 'Content',
+        })
+      : i18n.translate(
+          'xpack.enterpriseSearch.content.searchIndices.searchIndices.emptyPageTitle',
           {
-            defaultMessage: 'Search indices',
+            defaultMessage: 'Welcome to Enterprise Search',
           }
-        ),
-        rightSideItems: [createNewIndexButton],
-      }}
-    >
-      <EuiBasicTable items={searchIndices} columns={columns} />
-    </EnterpriseSearchContentPageTemplate>
+        );
+
+  return (
+    <>
+      <EnterpriseSearchContentPageTemplate
+        pageChrome={baseBreadcrumbs}
+        pageViewTelemetry="Search indices"
```

Comment:
question: do we have schema's or a strategy defined for this telemetry somewhere? I've seen bits and pieces, but no comprehensive document.

Reply:
I am not aware of it if there are any 🤷

Reply:
Seems like a thing we should work on getting then :)